README.md

```diff
@@ -173,6 +173,9 @@ server:
   # output `path` where a generated webhook kubeconfig will be stored.
   generateKubeconfig: /etc/kubernetes/heptio-authenticator-aws.kubeconfig # (default)
 
+  # role to assume before querying EC2 API in order to discover metadata like EC2 private DNS Name
+  defaultEC2DescribeInstancesRoleARN: arn:aws:iam::000000000000:role/DescribeInstancesRole
```

Comment:
Nit: the "default" part here is slightly confusing to me. Maybe something like `serverAssumeRoleARN`?

Reply:
Ah, default is incorrect now.  I like having EC2DescribeInstances in the name though, as I think it helps differentiate it from the other references to roles.  How about `serverEC2DescribeInstancesRoleARN`?

Reply:
Yeah I like that.